h/services/__init__.py

```diff
@@ -24,9 +24,11 @@ def includeme(config):
     config.register_service_factory('.groupfinder.groupfinder_service_factory', iface='h.interfaces.IGroupService')
     config.register_service_factory('.links.links_factory', name='links')
     config.register_service_factory('.list_groups.list_groups_factory', name='list_groups')
+    config.register_service_factory('.list_organizations.list_organizations_factory', name='list_organizations')
     config.register_service_factory('.nipsa.nipsa_factory', name='nipsa')
     config.register_service_factory('.oauth_provider.oauth_provider_service_factory', name='oauth_provider')
     config.register_service_factory('.oauth_validator.oauth_validator_service_factory', name='oauth_validator')
+    config.register_service_factory('.organization.organization_factory', name='organization')
```

Comment:
When you ultimately merge this PR, could you keep the commit for this (versus squashing and merging)? I don't really mind it being in this PR, but keeping the commit would be nice.

Reply:
will do!